descqa/shear_test.py

```diff
@@ -0,0 +1,233 @@
+
+#from __future__ import unicode_literals, absolute_import, division
+import os
+import numpy as np
+from .base import BaseValidationTest, TestResult
+from .plotting import plt
+
+
+import sys
+sys.path.insert(0, '/global/common/software/lsst/common/miniconda/py3-4.2.12/lib/python3.6/site-packages')
```

Comment:
remove both lines with `sys.path.insert` (this will be taken care by the framework)